Modules/vc-module-pricing/VirtoCommerce.PricingModule.Data/VirtoCommerce.PricingModule.Data.csproj

```diff
@@ -20,6 +20,7 @@
     <ItemGroup>
         <ProjectReference Include="..\..\..\VirtoCommerce.Platform.Core\VirtoCommerce.Platform.Core.csproj" />
         <ProjectReference Include="..\..\..\VirtoCommerce.Platform.Data\VirtoCommerce.Platform.Data.csproj" />
+        <ProjectReference Include="..\..\..\VirtoCommerce.Platform.Security\VirtoCommerce.Platform.Security.csproj" />
```

Comment:
Seems this will be redundant after removing requirement

Reply:
now we need this reference